explainer.md

```diff
@@ -1,6 +1,11 @@
-explainer.md
 # Async cookies API explained
 
+This is a proposal to bring an asynchronous cookie API to scripts running in HTML documents and [service workers](https://github.com/slightlyoff/ServiceWorker).
+
+[HTTP cookies](https://tools.ietf.org/html/rfc6265) have, since their [origins at Netscape (preserved by archive.org)](https://web.archive.org/web/0/http://wp.netscape.com/newsref/std/cookie_spec.html), provided a [valuable state-management mechanism](http://www.montulli-blog.com/2013/05/the-reasoning-behind-web-cookies.html) for the web. Unfortunately the synchronous single-threaded script-level interface to them - also originating in the Netscape browser -  [has been a source of complexity and performance woes](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Sep/0083.html) exacerbated by the move in many browsers from a single-process, single-threaded model without a general expectation of responsiveness for scripted event handling while processing cookie operations to the modern web which strives for smoothly responsive high performance in a multithreaded, multi-process environment where a cookie operation in one part of a web application cannot block the rest of the application, web origin, or the browser as a whole.
```

Comment:
The part beginning with "Unfortunately" is a very long sentence. Maybe break it up.